Orange/data/domain.py

```diff
@@ -58,69 +69,101 @@ def __init__(self, source, destination):
 
 
 class Domain:
-    def __init__(self, attributes, class_vars=None, metas=None, source=None):
+    def __init__(self, attributes=[], class_vars=[], metas=[], source=None):
```

Comment:
Please avoid using lists as default parameters. Even if it works in this case, it is a bad practice. If the value would be stored, all instances would share the **same** instance of the list.